sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SyntaxTrackers/CSharpElementAccessTracker.cs

```diff
@@ -63,18 +63,15 @@ public CSharpElementAccessTracker(IAnalyzerConfiguration analyzerConfiguration,
 
         public override ElementAccessCondition MatchProperty(MemberDescriptor member)
         {
-            return (context) => {
-                var accessSyntax = context.Expression as ElementAccessExpressionSyntax;
-                if(accessSyntax.Expression.IsKind(SyntaxKind.SimpleMemberAccessExpression))
-                {
-                    var memberAccessSyntax = accessSyntax.Expression as MemberAccessExpressionSyntax;
-                    var propertyName = memberAccessSyntax.Name.Identifier.ValueText;
-                    var enclosingClassType = context.SemanticModel.GetTypeInfo(memberAccessSyntax.Expression);
-                    return enclosingClassType.Type != null && member.IsMatch(propertyName, enclosingClassType.Type, false);
-                }
-                return false;
-            };
-
+            return (context) =>
```

Comment:
nitpick: you can replace the body of the method, as it has only 1 return
```
public override ElementAccessCondition MatchProperty(MemberDescriptor member) =>
	(context) =>
		// ... code
```

Reply:
Good to know :)